test/shopify.test.js

```diff
@@ -532,6 +532,12 @@ describe('Shopify', () => {
       );
     });
 
+    it('returns undefined when throttle is undefined', () => {
```

Comment:
I would prefer something like this https://github.com/MONEI/Shopify-api-node/blob/3.4.2/test/shopify.test.js#L712-L723 where `res.body.extensions.cost` is an empty object.

Reply:
This is testing the missing like `189` coverage, when no argument is given for `throttle`, the function escapes early.
```
Shopify.prototype.updateGraphqlLimits = function updateGraphqlLimits(throttle) {
  if (!throttle) return;

  const limits = this.callGraphqlLimits;

  limits.current = throttle.maximumAvailable - throttle.currentlyAvailable;
  limits.remaining = throttle.currentlyAvailable;
  limits.restoreRate = throttle.restoreRate;
  limits.max = throttle.maximumAvailable;

  this.emit('callGraphqlLimits', limits);
};
```
Writing the test as
```
it('returns undefined when throttle is undefined', () => {
  const shopify = new Shopify({ shopName, accessToken, timeout: 100 });

  return shopify.updateGraphqlLimits().then(() => {
     expect(shopify.callGraphqlLimits).to.deep.equal({});
  });
});
```
therefore fails because we can't call `then()` on `undefined`

Reply:
Write it as follow:

```js
it('does not update callGraphqlLimits if throttleStatus is missing', () => {
  scope.post('/admin/api/graphql.json').reply(200, {
    extensions: {
      cost: {}
    }
  });

  return shopify.graphql('query').then(() => {
    expect(shopify.callGraphqlLimits).to.deep.equal({
      restoreRate: 50.0,
      remaining: 997,
      current: 3,
      max: 1000.0
    });
  });
});
```

Reply:
Done 👍